build-config.js

```diff
@@ -21,6 +21,12 @@ const angularVersion = packageJson.requiredAngularVersion;
  */
 const mdcVersion = packageJson.requiredMDCVersion;
 
+/**
+ * Required rxjs version for all Angular Material packages. This version will be used
+ * as the peer dependency version for rxjs in all release packages that require rxjs.
+ */
+const rxjsVersion = packageJson.requiredRxjsVersion;
```

Comment:
What's the background on this being necessary? Historically we haven't added direct deps for RxJS because we stick to the same requirement as Angular itself. 

cc @devversion